manifest/js-Dog-CustomVision-manifest.json

```diff
@@ -0,0 +1,16 @@
+{
+    "$schema": "https://schemas.botframework.com/schemas/skills/v2.2/skill-manifest.json",
+    "$id": "DogCustomVision",
+    "name": "DogCustomVision",
+    "version": "1.0",
+    "description": "This is a sample skill for custom vision integration",
+    "publisherName": "CoSo",
+    "privacyUrl": "https://www.corporatesoftware.ch/privacy-policy-imprint",
+    "copyright": "Copyright (c) Corporate Software AG. All rights reserved.",
+    "license": "MIT",
+    "iconUrl": "<IMG_URL>",
+    "tags": [
```

Comment:
was bedeuten diese? wenn du es nicht weisst, nimm es raus, oder bringe es in erfahrung und passe die tags an.